jerry-core/parser/js/byte-code.h

```diff
@@ -617,6 +617,8 @@
  */
 typedef struct
 {
+  uint16_t size;                    /**< real size >> MEM_ALIGNMENT_LOG */
```

Comment:
why not embed an `ecma_compiled_code_t` in these structs instead of copying the structure?